### ISSUE TITLE
Rails 5.1 support

### ISSUE THREAD

**Issue by samstickland** [SchemaPlus/schema_plus_enums#8]
Hi,

Unless I need more coffee shouldn't the Rails 5.1 support specify `<5.2`. I currently get the following whilst trying to install with master

```
Bundler could not find compatible versions for gem "activerecord":
  In snapshot (Gemfile.lock):
    activerecord (= 5.1.3)

  In Gemfile:
    draper was resolved to 3.0.0, which depends on
      activemodel-serializers-xml (~> 1.0) was resolved to 1.0.1, which depends on
        activerecord (> 5.x)

    delayed_job_active_record was resolved to 4.1.2, which depends on
      activerecord (< 5.2, >= 3.0)

    flipper-activerecord was resolved to 0.1.1, which depends on
      activerecord (>= 4.2.0.beta2)

    immigrant was resolved to 0.3.6, which depends on
      activerecord (>= 3.0)

    immigrant was resolved to 0.3.6, which depends on
      activerecord (>= 3.0)

    immigrant was resolved to 0.3.6, which depends on
      activerecord (>= 3.0)

    immigrant was resolved to 0.3.6, which depends on
      activerecord (>= 3.0)

    immigrant was resolved to 0.3.6, which depends on
      activerecord (>= 3.0)

    immigrant was resolved to 0.3.6, which depends on
      activerecord (>= 3.0)

    paper_trail was resolved to 7.0.2, which depends on
      activerecord (< 5.2, >= 4.0)

    activeadmin-reform was resolved to 0.3.0, which depends on
      rails (>= 3.2) was resolved to 5.1.3, which depends on
        activerecord (= 5.1.3)

    rails-erd was resolved to 1.5.2, which depends on
      activerecord (>= 3.2)

    schema_plus_enums was resolved to 0.1.6, which depends on
      activerecord (<= 5.1, >= 4.2)

    state_machines-activerecord was resolved to 0.5.0, which depends on
      activerecord (< 5.2, >= 4.1)

    whiny_validation was resolved to 0.1.1, which depends on
      activerecord

    whiny_validation was resolved to 0.1.1, which depends on
      activerecord

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by patleb** [comment]
should be addressed by this [pull request](https://github.com/SchemaPlus/schema_plus_enums/pull/10)

**Comment by ronen** [comment]
PR merged, version 0.1.7 released.  Thanks.
